### PR TITLE
feat: added card component to the  library

### PR DESCRIPTION
**Pull request by nikhil-sharma-nks**
### preview  - https://deploy-preview-4--silk-ui.netlify.app/components/card
1. Vertical Card
2. Card with badge
3. Horizontal card
4. Cards with text overlay
5. Text only cards
6. Cards with shadow
7. Cards with dismiss
card-gif

### CODE REVIEW

File: components/card.html [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+            style="
```
Comment (chanchal-panpaliya) [inline components/card.html:150]: avoid inline style

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+.overlay-text {
+  color: var(--white);
+  position: absolute;
+  top: 10rem;
+  left: 5rem;
+}
+
+.card-shadow {
+  box-shadow: rgba(0, 0, 0, 0.4) 0px 30px 90px;
```
Comment (chanchal-panpaliya) [inline components/card.css:102]: use only rem or em units.

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
+@import url(./button.css);
+.card {
+  background-color: var(--white);
+  border: 1px solid #555;
```
Comment (chanchal-panpaliya) [inline components/card.css:4]: change border color #555 into var(--bordercolor). if you are using this color many times .

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good . need to change - units , avoid inline style , and color code .

File: components/card.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  height: 100%;
+  z-index: 3;
+  background-color: rgba(0, 0, 0, 0.5);
+  cursor: pointer;
+}
+
+.overlay-text {
+  color: var(--white);
+  position: absolute;
+  top: 10rem;
```
Comment (manujagobind) [inline components/card.css:97]: Will % make more sense here?

**Pull request merged**